kafka-delete-topic/capability_service/kafka.py

```diff
@@ -0,0 +1,183 @@
+    ):
+        self.environment = environment
+        self.message = message
+        super().__init__(self.message)
+
+    def __str__(self):
+        return f"{self.environment} -> {self.message}"
+
+
+class Topic:
+    """A class that represent a Kafka topic."""
+
+    def __init__(
+        self,
+        blaster_bearer_token: str,
+        blaster_topics_api_url: str,
+        blaster_cluster_uuid_prod: str,
+        blaster_cluster_uuid_dev: str,
+        confluent_env_id_prod: str,
+        confluent_env_id_dev: str,
+        confluent_cluster_id_prod: str,
+        confluent_cluster_id_dev: str,
+        level: int = logging.INFO,
+    ) -> None:
+        """Class constructor.
+        :param blaster_bearer_token: A Bearer token which can authenticate with the capability service.
+        :param blaster_topics_api_url: The full URL to the Topic API, for instance https://<hostname>/capability/api/v1/topics # noqa 501
+        :param blaster_cluster_uuid_prod: The UUID for the Kafka production cluster.
+        :param blaster_cluster_uuid_dev: The UUID for the Kafka development cluster.
+        :param confluent_env_id_prod: A Confluent Kafka environment id for production.
+        :param confluent_env_id_dev: A Confluent Kafka environment id for development.
+        :param confluent_cluster_id_prod: The ID for the Confluent Kafka production cluster.
+        :param confluent_cluster_id_dev: The ID for the Confluent Kafka development cluster.
+        :param level: A valid log level from the logging module. Default: logging.INFO
+        :type blaster_bearer_token: str
+        :type blaster_topics_api_url: str
+        :type blaster_cluster_uuid_prod: str
+        :type blaster_cluster_uuid_dev: str
+        :type confluent_env_id_prod: str
+        :type confluent_env_id_dev: str
+        :type confluent_cluster_id_prod: str
+        :type confluent_cluster_id_dev: str
+        :type level: int
+        """
+        logging.basicConfig(
+            format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=level
+        )
+        self.blaster_bearer_token: str = blaster_bearer_token
+        self.blaster_topics_api_url: str = blaster_topics_api_url
+        self.blaster_cluster_uuid_prod: str = blaster_cluster_uuid_prod
+        self.blaster_cluster_uuid_dev: str = blaster_cluster_uuid_dev
+        self.confluent_env_id_prod = confluent_env_id_prod
+        self.confluent_env_id_dev = confluent_env_id_dev
+        self.confluent_cluster_id_prod = confluent_cluster_id_prod
+        self.confluent_cluster_id_dev = confluent_cluster_id_dev
+        self.log_level: int = level
+
+    def _get_cluster_uuid(self, environment_name: str) -> str:
+        """
+        :param environment_name: A valid Kafka environment name. Currently only dev and prod clusters exist.
+        :type environment_name: str
+        :return: str
+        """
+        if environment_name.upper() in ["PROD", "PRODUCTION"]:
```

Comment:
It seems like you are doing the same logic over and over again here caused by the dublicated variables above which is dependent on environment.

Reply:
Now loading env variables from .env file through config class.